src/Entity/Suite.php

```diff
@@ -0,0 +1,367 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\SuiteRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: SuiteRepository::class)]
+class Suite
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\ManyToOne(inversedBy: 'suite')]
+    private ?Execution $execution = null;
+
+    #[ORM\Column(length: 50)]
+    private ?string $uuid = null;
+
+    #[ORM\Column(type: Types::TEXT)]
+    private ?string $title = null;
+
+    #[ORM\Column(length: 40, nullable: true)]
+    private ?string $campaign = null;
+
+    #[ORM\Column(length: 200, nullable: true)]
+    private ?string $file = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $duration = null;
+
+    #[ORM\Column(name: 'hasSkipped', nullable: true)]
+    private ?bool $hasSkipped = null;
+
+    #[ORM\Column(name: 'hasPending', nullable: true)]
+    private ?bool $hasPending = null;
+
+    #[ORM\Column(name: 'hasPasses', nullable: true)]
+    private ?bool $hasPasses = null;
+
+    #[ORM\Column(name: 'hasFailures')]
+    private ?bool $hasFailures = null;
+
+    #[ORM\Column(name: 'totalSkipped', nullable: true)]
+    private ?int $totalSkipped = null;
+
+    #[ORM\Column(name: 'totalPending', nullable: true)]
+    private ?int $totalPending = null;
+
+    #[ORM\Column(name: 'totalPasses', nullable: true)]
+    private ?int $totalPasses = null;
+
+    #[ORM\Column(name: 'totalFailures', nullable: true)]
+    private ?int $totalFailures = null;
+
+    #[ORM\Column(name: 'hasSuites', nullable: true)]
+    private ?int $hasSuites = null;
+
+    #[ORM\Column(name: 'hasTests', nullable: true)]
+    private ?int $hasTests = null;
+
+    #[ORM\Column(nullable: true)]
+    private ?int $parent_id = null;
```

Comment:
Actually, it's a suite. I add a note in my roadmap to improve it.

Reply:
Ah ok, then maybe in another PR you could change this with a `OneToOne` relation instead? This could also simplify you list of children suites that would be automatically filled instead of programmatically fill them in

Reply:
It's exactly what I want to do ;)